angular/projects/admin/src/app/admin/components/dialog/dialog.component.html

```diff
@@ -0,0 +1,40 @@
+<div [ngSwitch]="data.type">
+
+    <div *ngSwitchCase="'delete'">
+      <div class="dialog-title">
+        <h1 mat-dialog-title>Delete</h1>
+        <mat-icon color="warn">warning</mat-icon>
+      </div>
+      <div mat-dialog-content>
+        <p class="mat-body">Are you sure to delete the "{{data.title}}" ?</p>
+      </div>
+      <div mat-dialog-actions>
+        <button mat-button
+                (click)="closeDialog()">Cancel</button>
+        <button mat-button
+                color="warn"
+                (click)="delete()">Delete</button>
+      </div>
+    </div>
+
+    <div *ngSwitchCase="'detail-file'">
```

Comment:
I think this one can be specific for file related module. E.g. the media gallery.

Reply:
Resolve on this commit : https://github.com/oddbit/tanam/pull/115/commits/d404e994e139b9d5e810919a5d7cecf37ffd904b